learn/guides/publish-packages-to-ballerina-central.md

```diff
@@ -112,6 +112,22 @@ Now, that you are ready to publish, execute the command below to publish the pac
 bal push
 ```
 
+### Publish successive package versions 
```

Comment:
Shall we say "Release a newer version of a package"?

Reply:
How about "Publish a newer version of a package", since this guide seems to have used the word "publish" instead of using "release" in almost everywhere? :) @praneesha WDYT?

Reply:
Shall we say `Publish a new version of a package`?